tsconfig.json

```diff
@@ -1,25 +1,33 @@
 {
 	"compilerOptions": {
-		"lib": ["es5", "es6", "es2015", "es2016", "es2017", "es2018"],
+		"lib": [
+			"es2019",
```

Comment:
I think we don't need this when targeting es2019 or do we?

Reply:
I think we do: https://github.com/tsconfig/bases/blob/main/bases/node12.json